src/domains/ecommerce-app/05_functions.tf

```diff
@@ -0,0 +1,128 @@
+resource "azurerm_resource_group" "ecommerce_functions_rg" {
+  name     = format("%s-ecommerce-fn-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+# Subnet to host ecommerce function
+module "ecommerce_function_snet" {
+  source                                         = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v4.3.2"
+  name                                           = format("%s-ecommerce-fn-snet", local.project)
+  address_prefixes                               = [var.cidr_subnet_ecommerce_functions]
+  resource_group_name                            = azurerm_resource_group.ecommerce_functions_rg.name
+  virtual_network_name                           = data.azurerm_virtual_network.vnet.name
+  enforce_private_link_endpoint_network_policies = true
+
+  service_endpoints = [
+    "Microsoft.Web",
+  ]
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+module "ecommerce_function_app" {
```

Comment:
All the names should be refactored, to add `transactions` (for  exmpale `ecommerce-transactions-functions-fn`), because if we want to create a new ecommerce function, we will already have a generic function.

Reply:
Partially fixed in 42cebbaf (see other comment)